src/dms-preview/azext_dms/custom.py

```diff
@@ -33,15 +45,16 @@ def create_or_update_project(
     # Set inputs to lowercase
     source_platform = source_platform.lower()
     target_platform = target_platform.lower()
+    scenario_handled_in_core = core_handles_scenario(source_platform, target_platform)
 
     # Validation: Test scenario eligibility
-    if not determine_source_target_eligibility(source_platform, target_platform):
+    if scenario_handled_in_core:
```

Comment:
Instead of checking what scenarios core supports, can we check for what scenarios the extension supports? This way we don't have to keep the two connected but purely keep any scenario supported in extension independent from the ones in core.